features/bicycle/ref.feature

```diff
@@ -25,8 +25,8 @@ Feature: Bike - Way ref
             | ab    |      | E7  |
 
         When I route I should get
-            | from | to | route |
-            | a    | b  | E7,E7 |
+            | from | to | route                               | ref   |
+            | a    | b  | {highway:primary},{highway:primary} | E7,E7 |
```

Comment:
@emiltin what's your take on http://wiki.openstreetmap.org/wiki/Cycle_routes Shouldn't the bike profile support this?